portability-api/src/main/java/org/dataportabilityproject/webapp/StartCopyHandler.java

```diff
@@ -70,6 +76,11 @@ public void handle(HttpExchange exchange) throws IOException {
     // Valid job must be present
     String jobId = JobUtils.decodeId(encodedIdCookie);
 
+    // Validate XSRF token is present in request header. strip out any quotation marks that Angular adds and remove whitespace.
```

Comment:
Is there a doc or bug report about this in angular that we can refer to?

Reply:
I updated the comment. This is caused by HttpCookie.toString surrounding the cookie value in doublequotes. This is valid cookie syntax and we parse it correctly, however it causes the angular app to surround the header value in double doublequotes (the first of which gets stripped out with the initialization of the http-exchange).